### DISCUSSION TITLE
Is sentry mode messing up efficiency and consumption stats? or How to extract sentry mode consumption?

### DISCUSSION THREAD

**Discussion by benoitm974** [teslamate-org/teslamate#2441] [Q&A]
Hi,

I do park my car in a private street where I can plug it but since garbage truck uses the street I keep sentry mode on. I have teslamate running since day 1 and i was wondering how much energy this sentry mode has consumed since the beginning.

I'm seeking for advice as of is there report that would help on this (campire drain seems close but I'm not sur it does not count for other drain of power heating system, pre-conditioninf etc...)

or if there is a way to query all drives and multiply the efficiency per distance and then extract the consumed energy driving to then remove from total energy added to the tesla ?

Any thoughs ?

**Comment by cwanja** [comment]
On average, sentry mode is likely draining 5-10% in a 24 hour period. I am not sure anyone has ever measured this to be precise. But a measure would be to calculate the drain with and without sentry mode on, in as like conditions as possible. You could also drain 1% on sentry mode and see how long that took and then compare it to non-sentry mode losing 1%. It is not exactly noble prize winning science, but it’s a start.

**Comment by benoitm974** [comment]
@cwanja thanks for the answer, I was more looking for a query in Grafana that would be a good indicator of sentry spend energy. My belief is that in a case like me the amount of energy consume by sentry mode, is probably higher than my insurance deductible :( ... Plus bying an EV and consuming a significant amount of energy in filming is not very ecological...

**Reply by cwanja** [reply]
Since the API does not classify electricity spend of specific functions (HVAC, sentry, preconditioning battery, etc.) there is no one query that can compute that for you. You can get "ideal" spend based upon my previous suggestions (unless someone else knows something I do not - entirely possible).
